From 43bb2c3373aa5b90a03be61488898397f44f31b3 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Tue, 26 Mar 2019 15:25:56 +0100
Subject: [PATCH 087/386] arm64: dts: marvell: armada-cp110: disable ihb1 axi
 monitor

Since AXI monitors are used for debugging purpose disable it by default.
For debugging purpose it can be enabled separately per board or SoC.

Change-Id: Ic83a13fa9952b68a93225634634ce15084a95de3
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/6360
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index d9a81d1a9358..5b2d9662891a 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -233,6 +233,7 @@
 			clocks = <&CP110_LABEL(clk) 1 3>;
 			clock-names = "apb_pclk";
 			bus-width = <40>;
+			status = "disabled";
 			port@0 { };
 		};
 
@@ -242,6 +243,7 @@
 			clocks = <&CP110_LABEL(clk) 1 3>;
 			clock-names = "apb_pclk";
 			bus-width = <40>;
+			status = "disabled";
 			port@0 { };
 		};
 
-- 
2.17.1

